recipes/ffmpeg-python/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ffmpeg-python" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+    sha256: 65225db34627c578ef0e11c8b1eb528bb35e024752f6f10b78c011f6f64c4127
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - ffmpeg
+    - future
```

Comment:
this seems to be missing a lot of dependencies

Reply:
It is simply a wrapper based on FFmpeg process. So there is no dependency (see repository setup.py). Only pytest-runner 
as setup dependency and future as running dep